spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -416,9 +416,22 @@ New App Logs:
 2016-08-08T17:11:11.94-0700 [APP/0]      OUT 2016-08-09 00:11:11.941  INFO 26 --- [ foo.time.foo-1] TEST [log.sink                       : 08/09/16 00:11:11]
 ----
 
-Deleting the old version `foo-log` from the CF CLI would make all the payload consumed by the `logv2` application. Now,
+Deleting the old version `foo-log` from the CF CLI would make all the payload consumed by the `foo-log-v2` application. Now,
 you've successfully rolling upgraded an application in the streaming pipeline without bringing it down in entirety to do
```

Comment:
I think it reads better if you remove the word "rolling" from this line.  Otherwise, this is great.